src/actions/deleteUserActions.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs')
+
+let findUser = (users, id) => {
```

Comment:
Did you get the chance to sync with the other teams regarding `findUser` implementation?